
Beware that seabios 0.6.2 is broken for QXL. You need to apply this
patch

diff --git a/src/config.h b/src/config.h
index e6e07c9..cd71c3a 100644
--- a/src/config.h
+++ b/src/config.h
@@ -51,7 +51,7 @@
 #define BUILD_PCIPREFMEM_END      0
 #else
 #define BUILD_PCIMEM_START        0xf0000000
-#define BUILD_PCIMEM_SIZE         0x08000000    /* half- of pci window */
+#define BUILD_PCIMEM_SIZE         0x09000000    /* half- of pci window */
 #define BUILD_PCIMEM_END          (BUILD_PCIMEM_START + BUILD_PCIMEM_SIZE)
 #define BUILD_PCIPREFMEM_START    BUILD_PCIMEM_END
 #define BUILD_PCIPREFMEM_SIZE     (BUILD_PCIPREFMEM_END - BUILD_PCIPREFMEM_START)
